modules/standard/Random.chpl

```diff
@@ -2114,9 +2128,14 @@ module Random {
     :mod:`PCGRandom` and :mod:`NPBRandom` for RNGs that can be
     instantiated. To create a random stream, use :proc:`makeRandomStream`.
 
-    Note that whether or not 0.0 and 1.0 are possiblly generated real
-    numbers varies. (The can be generated by :mod:`PCGRandom`
-    but not by :mod:`NPBRandom`).
+    Note different implementations of this interface can vary in whether or not
+    they can generate 0.0 and 1.0.  (e.g. The can be generated by
```

Comment:
"The" -> "They".  While here, might change "and" into "and/or" (since we might conceive of future RNGs that generate one but not the other?)